plugins/resource-monitor-plugin/src/plugin.ts

```diff
@@ -27,6 +27,9 @@ export async function start(context: theia.PluginContext): Promise<void> {
 }
 
 export async function getNamespace(): Promise<string> {
+  if (process.env.DEVWORKSPACE_NAMESPACE !== undefined) {
```

Comment:
it looks like we're breaking abstraction of the che API and bypassing it using directly the underlying environment variables

getNamespace() should be a method of che.devfile or che.workspace API  and then implementation for che-server or devWorkspaces would be different

Reply:
@benoitf  I've added `getCurrentNamespace` method into `che.workspace` API. Could you please take a look at the changes again.

Reply:
thanks